protocol/x/revshare/keeper/revshare_test.go

```diff
@@ -332,23 +332,23 @@ func TestKeeper_GetAllRevShares_Valid(t *testing.T) {
 					},
```

Comment:
For my understanding, how did we set up the tier for `Bob` in this test case? Didn't find anything related in the `setUp()`

Reply:
bob address has coins already setup [here](https://github.com/dydxprotocol/v4-chain/blob/affan/update-revshare/protocol/testutil/constants/genesis.go#L771) for staking. This just puts bob at the max tier

Reply:
I see, could we document in the unit test? Not very obvious to reader